lib/python/bind_operators.h

```diff
@@ -246,6 +252,12 @@ void bind_binary_scalars(pybind11::class_<T, Ignored...> &c) {
   OpBinder<ScalarToVariable>::binary<int64_t>(c);
 }
 
+template <class T, class... Ignored>
+void bind_comparison_scalars(pybind11::class_<T, Ignored...> &c) {
+  OpBinder<ScalarToVariable>::comparison<double>(c);
+  OpBinder<ScalarToVariable>::comparison<int64_t>(c);
```

Comment:
Would it work to also support `bool`, or are there pybind11 overload resolution issues?

Reply:
Doesn't seem to work by just calling `OpBinder` with `bool`:
```.py
>       assert (sc.scalar(True) == True).value
E       scipp._scipp.core.DTypeError: 'equal' does not support dtypes 'bool', 'int64',
```

Also, what would be the point? PyCharm even complains about using the equality operator in this way because it violates PEP8 E712, i.e. don't compare bools using `__eq__`.

Reply:
👍